efficientdet/keras/efficientdet_keras.py

```diff
@@ -384,13 +385,14 @@ def __init__(self,
     self.data_format = data_format
     self.conv_ops = []
     self.bns = []
+    self.grad_checkpoint = grad_checkpoint
     if separable_conv:
       conv2d_layer = functools.partial(
           tf.keras.layers.SeparableConv2D,
           depth_multiplier=1,
           data_format=data_format,
-          pointwise_initializer=tf.initializers.VarianceScaling(),
-          depthwise_initializer=tf.initializers.VarianceScaling())
+          pointwise_initializer=tf.initializers.variance_scaling(),
+          depthwise_initializer=tf.initializers.variance_scaling())
```

Comment:
It makes the project incompatible with TensorFlow 2.1.0:
```
AttributeError: module 'tensorflow_core.python.keras.api._v2.keras.initializers' has no attribute 'variance_scaling'
```

Reply:
/cc @fsx950223

Reply:
Update to tf2.3

Reply:
@fsx950223, `requirements.txt` should be updated this way: https://github.com/google/automl/blob/3dd89c4b2443e8b8037018ff805edc1aa09fa6ee/efficientdet/requirements.txt#L7

Reply:
> @fsx950223, `requirements.txt` should be updated this way:
> 
> https://github.com/google/automl/blob/3dd89c4b2443e8b8037018ff805edc1aa09fa6ee/efficientdet/requirements.txt#L7

Yeah, https://github.com/google/automl/pull/846

Reply:
Thank you!